### PR TITLE
subtask:somn-30->create_dto_layer

### PR DESCRIPTION
**Pull request by evakule**

### CODE REVIEW

File: src/main/java/com/somn/controller/AccountController.java [hunk 1]
```diff
@@ -23,41 +25,47 @@
   private AccountEntityService accountEntityService;
   
   @RequestMapping(value = "api/v1/accounts", method = RequestMethod.GET)
-  public ResponseEntity<List<AccountEntity>> getAllAccounts() {
-    return accountEntityService.getAllAccounts().map(ResponseEntity::ok)
-        .orElseGet(ResponseEntity.status(HttpStatus.NOT_FOUND)::build);
+  public ResponseEntity<List<AccountDTO>> getAllAccounts() {
+    List<AccountDTO> accountDTOList = accountEntityService.getAllAccounts();
+    if (accountDTOList == null) {
```
Comment (ssuchkov) [inline src/main/java/com/somn/controller/AccountController.java]: please also check whether list has 0 size for all such cases
its better use null safe utility method (like apache commons provides)

File: src/main/java/com/somn/dto/AccountDTO.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```
Comment (ssuchkov) [inline src/main/java/com/somn/dto/AccountDTO.java]: Are you using builder somewhere?

Comment (evakule) [reply src/main/java/com/somn/dto/AccountDTO.java]: Fixed

File: src/main/java/com/somn/controller/AccountController.java (hunk 1 above)
Comment (evakule) [reply src/main/java/com/somn/controller/AccountController.java]: Done

**Pull request merged**